dev-haskell/commonmark/commonmark-0.1.1.4.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Pure Haskell commonmark parser"
+HOMEPAGE="https://github.com/jgm/commonmark-hs"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-haskell/parsec:=[profile?]
+	dev-haskell/text:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.2.0.1
+	test? ( dev-haskell/quickcheck
```

Comment:
I'd really do a new line here and before the last )

Reply:
This ebuild is generated using `hackport` - I can manually make these stylistic changes, but more appropriate would probably be to update the `hackport` thing so it outputs the stylings we want.

For example, I recently updated `hackport` to exclude `IUSE=""` from the ebuild.

Reply:
Yeah, let's not worry for now then (and yay!)

Reply:
Opened [an issue][https://github.com/gentoo-haskell/hackport/issues/92] on the hackport project so I don't forget about this.